### ISSUE TITLE
PHP7 compatibility

### ISSUE THREAD

**Issue by slaFFik** [woocommerce/woocommerce-gateway-paypal-express-checkout#174]
We ran a `php7cc` tool and found this:

```
File: .../plugins/woocommerce-gateway-paypal-express-checkout/includes/class-wc-gateway-ppec-payment-details.php
> Line 35: Indirect variable, property or method access
    $this->{$map[$index]};
```

Can you please check, that it doesn't produce any issues?

**Comment by gedex** [comment]
@slaFFik Thanks for the report. We'll take a look.

**Comment by gedex** [comment]
Closing this as running `php7cc` against `master` doesn't produce any warnings.
